ProteoformSuiteInternal/CytoscapeScript.cs

```diff
@@ -186,9 +188,11 @@ private static string get_script(int feature_count, bool quantitative, string ed
         {
             string tsv_header = "accession_1\t" + lysine_count_header + "\taccession_2\t" + delta_mass_header + "\t" + edge_ptm_header;
             string edge_rows = "";
-            foreach (ProteoformRelation r in families.SelectMany(f => f.relations))
+            foreach (ProteoformRelation r in families.SelectMany(f => f.relations).Distinct())
```

Comment:
I'm still a little baffled how two families can have the same relations, since they should be exclusive to each family. The relations are unique within each family (using a HashSet). I have seen a few duplicates, though... Maybe the hash setting is flawed.